plugins/certs/commands

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+source "$(dirname $0)/functions"
+
+case "$1" in
+  help | certs:help)
+    cat && cat<<EOF
+    certs:add <app> CRT KEY, Add an ssl endpoint to an app.
+    certs:generate <app> DOMAIN, Generate a key and certificate signing request (or self-signed certificate)
+    certs:info <app>, Show certificate information for an ssl endpoint.
+    certs:key <app> CRT KEY [KEY ...], Print the correct key for the given certificate.
+    certs:remove <app>, Remove an SSL Endpoint from an app.
+    certs:update <app> CRT KEY, Update an SSL Endpoint on an app.
+EOF
```

Comment:
For sure. I meant to add a task to redo the entirety of the docs for this; including the "public functions convention.